srcpkgs/session-desktop/template

```diff
@@ -0,0 +1,46 @@
+# Template file for 'session-desktop'
+pkgname=session-desktop
+version=1.6.9
+revision=1
+# discontinued Electron 32-bit support: https://www.electronjs.org/blog/linux-32bit-support
+# aarch64 don't seems very stable for now and isn't officialy supported: https://github.com/oxen-io/session-desktop/issues/1635
+archs="x86_64"
```

Comment:
I think the consensus is to not restricting archs. Instead use a `broken` statement (with comment). You can grep for that in the repository.

Also typo : `aarch64 doesn't seem very stable for now and isn't officially supported`

Reply:
I think this is fine, but is it also broken on x86_64-musl?

Reply:
Electron doesn't official support musl, there are patches but so it would to edit the builds scripts of Session, so it fetches the patched version of Electron, and I don't know how to do that (I could search tho)

Reply:
We have some Electron packages (Element, Vscode, RockerChat). You can look into those for the way to make the package use the system Electron.

Reply:
They use Electron 8.2.0, which is over 1y old...
( https://github.com/oxen-io/session-desktop/blob/v1.6.9/yarn.lock#L3351
 https://github.com/electron/electron/releases/tag/v8.2.0 )

So using system electron isn't a option right now (We do not, and never will ship electron 8, it is long EOL and I would rather not add a package which depends on it.

Reply:
> They use Electron 8.2.0, which is over 1y old...
> ( https://github.com/oxen-io/session-desktop/blob/v1.6.9/yarn.lock#L3351
> https://github.com/electron/electron/releases/tag/v8.2.0 )
> 
> So using system electron isn't a option right now (We do not, and never will ship electron 8, it is long EOL and I would rather not add a package which depends on it.

The app seems to run fine using electron 12 from what i'm testing rn ^^

Reply:
So I have spent a few hours trying to make it work with musl, but didn't get very far ^^' 
I guess it'd require some patching (main problem is yea the fact it uses an old electron version. Also, just that I wasn't able to make the install script not download electron and doing AppImage *which don't seems to work on musl* despite heavily modifying it, but about that I may just have done something wrong tho ^^').
Besides, signal do not either support musl, or Session is a fork of it.